src/components/ConfirmationPage.tsx

```diff
@@ -51,12 +54,13 @@ function ConfirmationPage({
     headingStyle,
     illustrationStyle,
     descriptionStyle,
+    containerStyle,
 }: ConfirmationPageProps) {
     const styles = useThemeStyles();
     const isLottie = isIllustrationLottieAnimation(illustration);
 
     return (
-        <>
+        <View style={containerStyle}>
```

Comment:
This wrapping View made the content not taking all the available space. Coming from https://github.com/Expensify/App/issues/58342